packages/core-mobile/app/services/wallet/WalletService.tsx

```diff
@@ -622,14 +639,20 @@ class WalletService {
     Logger.info('validating burned amount')
 
     const avalancheProvider = await NetworkService.getAvalancheProviderXP(
-      Boolean(network.isTestnet)
+      Boolean(network.isTestnet),
+      isDevnet(network)
     )
 
     const { isValid, txFee } = utils.validateBurnedAmount({
       unsignedTx,
       context: avalancheProvider.getContext(),
       baseFee: evmBaseFeeInNAvax,
-      feeTolerance: EVM_FEE_TOLERANCE
+      feeTolerance: EVM_FEE_TOLERANCE,
+      upgradesInfo: isDevnet(network)
+        ? await new info.InfoApi(network.rpcUrl)
+            .getUpgradesInfo()
+            .catch(() => undefined)
+        : undefined
```

Comment:
let's extract this to a variable. also what will happen when we pass undefined upgradesInfo to validateBurnedAmount?

Reply:
this is only undefined when etna is not activation, because the api is only available in etna.